stockroom/storages/generic.py

```diff
@@ -0,0 +1,30 @@
+from .. import parser
+from ..repository import StockRepository
+
+
+# TODO: Maybe need another name than `GenericStore` but keeping it-
+# since scope of the name MetricStore seems to be too narrow
+
+class GenericStore:
+    """
+    A generic data storage class that does the functionality of
+    hangar metadata
+    """
+    def __init__(self, write):
+        self._write = write
+        self._repo = StockRepository()
+
+    def save(self, key, value):
+        co = self._repo.checkout(write=True)
+        co.metadata[parser.generic_metakey(key)] = value
+        co.close()
```

Comment:
Use a context manager so that if some exception is thrown in the hangar metadata operation, then the checkout is still closed.

Reply:
handeld automatically if https://github.com/tensorwerk/stockroom/pull/1#discussion_r361956040 is implemented.